recipes/SimpleASTParserLog/src/Main.purs

```diff
@@ -137,12 +137,12 @@ parseAtom = do
       void $ string "." -- decimal point
       decimalsAsString <- parseNumSequence
       let fullString = digitsAsString <> "." <> decimalsAsString
-      case readFloat fullString of
-        x | isFinite x -> pure $ LitNum x
+      case Number.fromString fullString of
+        Just x | Number.isFinite x -> pure $ LitNum x
```

Comment:
Couldn't find the partial `readFloat` here, so replaced with `fromString`. Since there was already a `fromString` imported from `Data.Int`, I had to namespace both imports to disambiguate.

Reply:
The `Number.isFinite` check is no longer necessary as `Number.fromString` [already](https://github.com/purescript/purescript-numbers/blob/v8.0.0/src/Data/Number.purs#L53) handles [that](https://github.com/purescript/purescript-numbers/blob/v8.0.0/src/Data/Number.js#L12-L19).

Reply:
0668a5d